tests/python_package_test/test_sklearn.py

```diff
@@ -35,6 +42,9 @@
     softmax,
 )
 
+SKLEARN_MAJOR, SKLEARN_MINOR, *_ = _sklearn_version.split(".")
+SKLEARN_VERSION_GTE_1_6 = int(SKLEARN_MAJOR) >= 1 and int(SKLEARN_MINOR) >= 6
```

Comment:
This check won't work for versions >=2.0.

```suggestion
SKLEARN_VERSION_GTE_1_6 = (int(SKLEARN_MAJOR), int(SKLEARN_MINOR)) >= (1, 6)
```

Reply:
ah!!!! thank you for catching this mistake 🙈